index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+
+</head>
+
+<body>
+    <!-- <h1>Hello World, we are software developers</h1> -->
+    <header id="header">
+        <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo" >
+        <nav id="nav-bar">
+            <a class="nav-link" href="#premium">Features</a>
+            <a class="nav-link" href="#fast">How It Works</a>
+            <a class="nav-link" href="#quality">Pricing</a>
+        </nav>
+    </header> 
+    <main class="landingPage">
+        <h1> Handcrafted, home-made masterpieces</h1>
+
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input id="email" name="email" type="email" class="email" placeholder="Enter your email address" required>
+            <input id="submit" type="submit" class="submit" >
+    </form> 
+</main>
+
+    <section >
+    <article id="premium">
+        <i class="fa fa-3x fa-fire" style="color:orange;"></i>
+        <div class="article-bar">
+            <h1>Premium Materials</h1>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </div>
+    </article>
+
+    <article id="fast">
+        <i class="fa fa-3x fa-truck" style="color:orange;"></i>
+        <div  class="article-bar"> 
+            <h1>Fast Shipping</h1>
+            <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        </div>
+    </article>
+
+    <article id="quality">
+        <i class="fa fa-3x fa-battery-full" style="color:orange;"aria-hidden="true"></i>
+        <div  class="article-bar">
+            <h1><a href="#pricing"></a>Quality Assurance</a></h1>
```

Comment:
- [ ] Please do make sure you don't have more than one h1 tag for each page because having multiple make your code less semantic and confuse the search engine when searching for your page